tests/sync/test_browsertype_connect.py

```diff
@@ -12,12 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import time
+
 import pytest
 
 from playwright.sync_api import BrowserType, Error
 from tests.server import Server
 
 
+def test_browser_type_connect_slow_mo(
+    server: Server, browser_type: BrowserType, launch_server
+):
+    remote_server = launch_server()
+    t1 = time.monotonic()
+    browser = browser_type.connect(remote_server.ws_endpoint, slow_mo=100)
+    browser_context = browser.new_context()
+    page = browser_context.new_page()
+    assert page.evaluate("11 * 11") == 121
+    page.goto(server.EMPTY_PAGE)
+    browser.close()
+    assert (time.monotonic() - t1) > 3.8
```

Comment:
3.8 seems like it would result in flakyness, what about testing it just between the evaluate function instead of the page.goto, close etc.?

Reply:
It would not flake as I have counted the number of sleep call and it sums up to 3.8 also there is greater than so it won't